source/_posts/2021-04-09-android-april-2021-release.markdown

```diff
@@ -0,0 +1,71 @@
+---
+title: "Home Assistant Companion Android App April 2021 Release"
+description: "Search/Filter, In-App Log Viewer & Shortcuts in the April 2021 Android Release"
+date: 2021-04-07 00:00:00
+date_formatted: "April 9, 2021"
+comments: true
+author: Daniel Shokouhi
+categories: Release-Notes
+og_image: /images/blog/2021-04-09-android-april-2021/Companion.png
+---
+
+Hey Everyone! Its time for the April 2021 Android release. Last month we wrote that we would be aligning our releases closer to Home Assistant core so here we are! Another month went by and the Android app has started to see more contributors coming along. Thank you for everyone who has come by this month to contribute to all the awesome features this month. Hopefully you will join us next month as we can always use more contributors to move forward faster.
+
+## In-App Log Viewer
+
+This release has a new feature that will make viewing and sharing the logs MUCH easier. Some of you may recall the large amount of steps needed to get Android logs when you open issues. This usually involved installing [Android Studio](https://developer.android.com/studio) or another app and having to plug your device into a computer to grant some special ADB permissions. This was very difficult to get setup for the average user and just wasn't user friendly.
+
+Now when you navigate to App Configuration you will see a new Show and Share Logs section that will show all the logs from our app. This will include debug logs that we may have placed to help you troubleshoot issues such as location not updating or if the app has used the correct URL. You are not only able to view this log but also share and select from it so you can copy the logs to a [GitHub issue](https://github.com/home-assistant/android/issues/new?assignees=&labels=bug&template=Bug_report.md&title=) for our team to look into.
```

Comment:
```suggestion
We've added a new option to the app configuration called "Show and Share Logs". It will show all the logs from our app, including debug logs to help troubleshoot issues such as when the location does not update or if the app is using the correct URL. You are not only able to view this log but also share and select from it so you can copy the logs to a [GitHub issue](https://github.com/home-assistant/android/issues/new?assignees=&labels=bug&template=Bug_report.md&title=) for our team to look into.
```